components/molecule/notification/README.md

```diff
@@ -0,0 +1,41 @@
+# MoleculeNotification
+
+> Notifications offer users information on the system. The content may confirm that an action has been performed correctly, warn the user of an error or simply give information on certain circumstances.
+
+## Installation
+
+```sh
+$ npm install @schibstedspain/sui-molecule-notification --save
```

Comment:
We established that new components should be published to @s-ui org.

`@s-ui/react-molecule-notification`
Right @jcger ? Is it written somewhere. Otherwise tell me and I can do it in the wiki.